rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+# Lightweight RFC Process
+
+**Geode-RFC-0**
+
+**To be Reviewed By:** June 20th 2019
+
+**Authors:** amurmann@apache.org
+
+**Status:** **Active** | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+With some of our recent efforts, we have seen consensus on design decisions take a very long time. In order to be able to keep moving forward as a successful project, we need to be able to get to consensus faster and ultimately get to execution faster. Therefore I propose a slightly more formal process for us to get to consensus.
+
+We already have gotten pretty good at sharing proposals with each other. Not much should need to change with that. It’s mainly the process around it that needs to be ironed out. This document will refer to that process as the “RFC process”.
```

Comment:
As this is the first mention of RFC, it would be nice to include what that stands for, to help out new contributors.